importer/helper.go

```diff
@@ -252,7 +252,9 @@ func updateAccounting(db idb.IndexerDb, genesisJSONPath string, numRoundsLimit i
 	}
 
 	lastlog := time.Now()
-	act := accounting.New()
+	cache, err := db.GetDefaultFrozen()
```

Comment:
if we keep a persistent default frozen map, accounting.go will update it on acfg

Reply:
Oops. Yeah I didn't mean to initialize this for every round.